docs/dev/guide/contract-verification.md

```diff
@@ -2,7 +2,7 @@
 
 ## What is contract verification?
 
-The bytecode generated by the Solidity compiler is essentially deployed when a Smart Contract is deployed on a network. The original Solidity code is not retained in any way.
+When you deploy a Smart Contract on a network, you’re essentially deploying the bytecode produced by the Solidity compiler. The original Solidity code is not retained in any way.
 
 Verifying a contract entails determining whether the on-chain bytecode generated from the Smart Contract source code complies. If it does, we can state that we have confirmed the bytecode's source code's integrity. Hence the name – verification.
```

Comment:
I don't understand this part? What do you mean compiles?
Verification makes sure that some external code (e.g. from GitHub) is the same code as the deployed code

Reply:
Rephased to this 

Contract verification is ensuring the correctness, security, and safety of the smart contract implementations which allows smart contract developers to prove and publish the source code of the contracts deployed on-chain.